src/prof_data.c

```diff
@@ -492,17 +492,24 @@ prof_dump_open(bool propagate_err, const char *filename) {
 	return fd;
 }
 
+#ifdef JEMALLOC_JET
+prof_dump_write_file_t *JET_MUTABLE prof_dump_write_file = malloc_write_fd;
```

Comment:
Do you mean that these file pointers should always be exposed, regardless of whether the environment is `JEMALLOC_JET` or not? For that, the corresponding declarations in the headers files will also be unconditionally exposed.

Reply:
Yeah, just exposing everything is fine (everything still gets name-mangled, so it's not like, exposed to the user).

In the long-run, it would help us on a few axes to be able to kill off the test/non-test variants entirely. That probably won't be totally possible without C++/some templatizing, but in the meantime we at least don't want to make things harder in that dimension.